### PR TITLE
Balance Changes for Kalimdor

### PR DESCRIPTION
**Pull request by Ripzss**
Bladestorm damage for Grom/Garrosh Nerfed by 10%.
Chen Drunken haze now got same values as Muradin
Voodoo Restoration Range nerfed by 10%
Grommash Hellscream Xp reward nerfed by 20%
Druid of the Talon Gold cost 14 -> 16
Druid of the Talon Hp increased 350 -> 390
Mountain Giant base armor 5 -> 6
Sentinels Archer Hp increased from 525 -> 560
Sentinels Archer base damage increased from 23 -> 25
HippoGryph Rider Base damage increased from 19 -> 20
HippoGryph Rider Base hp increased from 850 -> 900
2nd level added to Moon Glaive Locked Behind quest "Drive Them Back"

### CODE REVIEW

**Comment by YakaryBovine** [comment]
I'll review this properly later, but I can see one thing straight away: you need to update the description for Drive Them Back to indicate the new reward.

**Comment by Ripzss** [comment]
Done something wrong with the Moon Glaive level 2 research gonna fix this first

File: src/WarcraftLegacies.Source/Quests/Sentinels/QuestSentinelsKillFrostwolf.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public sealed class QuestSentinelsKillFrostwolf : QuestData
       "The Frostwolves have been ousted from Kalimdor, along with their Tauren allies. They will not be missed.";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => "Moon Riders are trainable at the Roost";
+    protected override string RewardDescription => "Moon Riders are trainable at the Roost, Moon Glaive second level is now reserachable at the War Academy";
```
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Quests/Sentinels/QuestSentinelsKillFrostwolf.cs:31]:
```suggestion
    protected override string RewardDescription => $"Learn to train {GetObjectName(Constants.UNIT_E022_MOON_RIDER_SENTINELS)}s from the {GetObjectName(Constants.UNIT_EDOS_ROOST_SENTINELS)}s and research {GetObjectName(Constants.UPGRADE_REMG_UPGRADE_MOON_GLAIVE_LIGHT_BLUE_RESEARCH)}'s second level from the {GetObjectName(Constants.UNIT_E00L_WAR_ACADEMY_SENTINEL_BARRACKS)}";
```

File: mapdata/WarcraftLegacies/UpgradeData/Skin/1735222610.json [hunk 2]
```diff
@@ -19,6 +19,48 @@
...
+      "Level": 2,
+      "Id": 829453415,
+      "Type": 3,
+      "Value": "Upgrade Moon Glaive [|cffffcc00Level 2|r]"
+    },
+    {
+      "Level": 2,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Causes Huntress attacks to strike an additional unit"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UpgradeData/Skin/1735222610.json:57]: Huntress should be in the possessive, plural form. Upgrade descriptions are full sentences so should end with a full stop. Thus: 
```suggestion
      "Value": "Causes Huntresses' attacks to strike an additional unit."
```

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Couple things to fix, but looks solid otherwise.

**Review by YakaryBovine — APPROVED** [review]
Looks good. Thanks for the solid attention to detail on keeping the tooltip numbers up to date.

**Pull request merged**